src/bootstrap/src/core/build_steps/check.rs

```diff
@@ -354,19 +354,22 @@ impl Step for CodegenBackend {
 
         cargo
             .arg("--manifest-path")
-            .arg(builder.src.join(format!("compiler/rustc_codegen_{backend}/Cargo.toml")));
+            .arg(builder.src.join(format!("compiler/rustc_codegen_{}/Cargo.toml", backend.name())));
```

Comment:
Suggestion: *maybe* a helper to construct the cg backend crate name? I.e. the `rustc_codegen_{}` interpolation happens *quite* often yet feels a bit magical.

Reply:
I suspect this *might* be useful for someone working on a new cg backend...? But yeah...